### PR TITLE
Issue #2331 expand ~ to homedir in source path while adding sshfs hostfolder interactively

### PR DESCRIPTION
**Pull request by anjannath**
Fiexs: #2331 

Not sure if we want to show it expanded or keep it as `~` in the o/p of command `minishift hostfolder list`

### CODE REVIEW

File: cmd/minishift/cmd/hostfolder/add.go [hunk 1]
```diff
@@ -127,6 +128,8 @@ func addHostFolder(cmd *cobra.Command, args []string) {
 
 func addSSHFSInteractive(manager *hostFolderConfig.Manager, name string) error {
 	source := util.ReadInputFromStdin("Source path")
+	source, err := homedir.Expand(source)
+	fmt.Println(source, err)
```
Comment (praveenkumar) [inline cmd/minishift/cmd/hostfolder/add.go]: I think this line only for test purpose and also you need to handle this err, mostly return if not `nil`.

Comment (anjannath) [reply cmd/minishift/cmd/hostfolder/add.go]: Yes..i was testing.. sorry forgot to put the WIP

**Comment by anjannath** [comment]
retest this please

**Comment by LalatenduMohanty** [comment]
> Not sure if we want to show it expanded or keep it as ~ in the o/p of command minishift hostfolder list

I like the current behavior. 

```
$ minishift hostfolder add myshare --source ~/myshare --target /mnt/sda1/myshare --type sshfs
Lalatendus-MacBook-Pro:minishift lmohanty$ minishift hostfolder list
Name      Type    Source                    Mountpoint          Mounted
myshare   sshfs   /Users/lmohanty/myshare   /mnt/sda1/myshare   N
```

File: Gopkg.lock [hunk 2]
```diff
@@ -411,6 +411,11 @@
   revision = "cdc0f941c4d0e0e94d85348285568d921891e138"
   version = "v2.1.1"
 
+[[projects]]
+  name = "github.com/mitchellh/go-homedir"
+  packages = ["."]
+  revision = "b8bc1bf767474819792c23f32d8286a45736f1c6"
```
Comment (LalatenduMohanty) [inline Gopkg.lock]: 👍

File: Gopkg.toml [hunk 3]
```diff
@@ -179,3 +179,7 @@ ignored = ["github.com/Sirupsen/logrus"]
   name = "github.com/pkg/sftp"
   version = "1.0.0"
 
+[[constraint]]
+  name = "github.com/mitchellh/go-homedir"
+  revision = "b8bc1bf767474819792c23f32d8286a45736f1c6"
```
[the remark this answers is not in the document]
Comment (LalatenduMohanty) [reply Gopkg.toml]: 👍

**Pull request merged**